tests/clone/test_server_installer.py

```diff
@@ -17,7 +18,7 @@
 test_log_path = os.path.join(user_config_dir(), 'tests_logs')
 test_log_file = os.path.join(user_config_dir(), 'tests_logs/plebnet.logs')
 test_child_file = os.path.join(user_config_dir(), 'test_child_config.cfg')
-test_child_DNA_file = os.path.join(user_config_dir(), 'Child_DNA.json')
+test_child_QTable_file = os.path.join(user_config_dir(), 'QTable.json')
```

Comment:
@HusainKapadia I am not 100% sure but shouldn't it be ChildQTable.json instead of QTable.json ?

Reply:
changed to proper name

Reply:
Yes agree